frontend/src/app/_components/room-link-host/room-link-host.component.html

```diff
@@ -0,0 +1 @@
+<a href="{{environment.backend}}/schedule/{{id}}/zoomHost" target="_blank" *ngIf="id && isZoomHost">Join Meeting as Host</a>
```

Comment:
Currently coded as `zoomhost`, but I'm not sure that matters.  (And I can trivially fix it at my end if it does.)

Reply:
will fix